game/scripts/npc/abilities/slark_dark_pact.txt

```diff
@@ -52,7 +52,7 @@
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "total_damage"                                    "75 150 225 300 650 1000"
+        "total_damage"                                    "150 250 350 450 950 1450" //OAA
```

Comment:
Whenever I see this, first thing that comes to mind is **Suicidal Slark** because he deals half of damage to himself. It will be very funny if spell amp eblade slark becomes a thing because of this, lmao. Does he need this change? no he doesnt. Essence Shift change alone is a big change.

Reply:
ok